src/GravitationalClustering.js

```diff
@@ -1,38 +1,21 @@
 import Random from 'random-js';
-
-var Particle = require('./Particle');
-var UnionFind = require('ml-disjoint-set');
-var Distance = require('ml-distance').distance;
-var randomInt = require('./Utils').randomInt;
+import ParticleCluster from './ParticleCluster';
+import Particle from './Particle';
+import UnionFind from 'ml-disjoint-set';
+import {distance as Distance} from 'ml-distance';
```

Comment:
this is not a good solution because if we want to save the model you cannot save the function, at least that you use something with eval that I don't like too much... I don't know what do you think @targos?